### DISCUSSION TITLE
能否复现橙子大佬3.10版本那个用户界面

### DISCUSSION THREAD

**Discussion by ATMfeng** [ywmoyue/biliuwp-lite#881] [feature]
### 问题描述

之前那个版本跟手机界面比较接近，用起来感觉舒服一点

### 描述解决方案

如果能搞的话就好了

**Comment by ywmoyue** [comment]
我试着改过双页模式，理论上可以做

但是遇到个问题，V3和V4的播放视频的操作逻辑不同：
V3是点击视频先进入详情页再点击分P进入播放器页面，在双页模式中右侧单独显示视频详情或播放器都是合理的
V4是点击视频直接进入包含播放器的详情页，在双页模式中右侧同时显示视频详情包括互动栏和播放器，播放器空间就被压缩了，用起来很奇怪
之后我就没再继续深入了
